Tools/Marabou.md

```diff
@@ -41,7 +41,9 @@ Repository (unsure what the difference is): https://github.com/NeuralNetworkVeri
 Project page: https://neuralnetworkverification.github.io/
 
 #### Last commit date:
-13 September 2020
+Marabou: 29 Nov 2022 (default branch)
+Marabou: 24 Sep 2021 (default branch)
```

Comment:
Would it be possible to add the username before the repo if there are multiple repos?
E.g. this would become:
```
NeuralNetworkVerification/Marabou: 29 Nov 2022 (default branch)
guykatzz/Marabou: 24 Sep 2021 (default branch)
29 November 2022 (last activity)
```

Reply:
Yes. This is possible. I will make a commit for this today or tomorrow without conflicts